spec/classes/puppet_agent_configure_spec.rb

```diff
@@ -8,8 +8,7 @@
       'class puppet_agent::params { $config = "/etc/puppetlabs/puppet/puppet.conf" }',
       'class puppet_agent { $config = $::test_config }',
       'include puppet_agent::params',
-      'include puppet_agent',
-    ]
+      'include puppet_agent']
```

Comment:
Weird that rubocop chose to close the square bracket here. Maybe if there is a new line after `[` it will want `]` on a seperate line also?

Reply:
I think it depends on how the first line of array looks like, will try to fix that